proto/osmosis/gamm/v1beta1/pool.proto

```diff
@@ -107,6 +107,11 @@ message PoolParams {
     (gogoproto.moretags) = "yaml:\"smooth_weight_change_params\"",
     (gogoproto.nullable) = true
   ];
+  google.protobuf.Timestamp start_time = 4 [
+    (gogoproto.stdtime) = true,
+    (gogoproto.nullable) = false,
```

Comment:
I think this should be nullable

Reply:
I'd prefer nullable, and make the chain default the start time to the current block time. Otherwise it will be hard to debug the odd start times in the future.